addons/website_forum/populate/forum_post.py

```diff
@@ -0,0 +1,42 @@
+from random import choices
+from odoo import models
+from odoo.tools import populate
+
+QUESTION_SIZES = { 'small': 100, 'medium': 1000, 'large': 90000 }
+MAX_REPLY_NUM = 7
+POST_SIZES = { key: size * (MAX_REPLY_NUM + 1) for key, size in QUESTION_SIZES.items() }
+QA_WEIGHTS = { '0': 25, '1': 35, '2': 20, '3': 10, '4': 4, '5': 3, '6':2, '7': 1 }
+ANSWERS_RANGE = list(range(0, (MAX_REPLY_NUM + 1)))
+WEIGHTS = list(QA_WEIGHTS.values())
+
+class Post(models.Model):
+    _inherit='forum.post'
+    _populate_sizes= POST_SIZES
```

Comment:
It's really time to switch your linter ;)
(There are many more styling issues below)

```suggestion
    _inherit = 'forum.post'
    _populate_sizes = POST_SIZES
```